src/components/hexString.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+import { shortenHex } from '../utils';
+
+const { Fragment } = React;
+
+interface HexStringProps {}
+
+const HexString: React.SFC<HexStringProps> = ({ children }) => {
+  return (
+    <Fragment>
+      <span className="leap-hex-full">{children}</span>
+      <span className="leap-hex-short" title={children as string}>
```

Comment:
maybe use `react-responsive` here? We already have it bundled

Reply:
will make things cleaner. However, I'm not aware of performance downside

I would then use three media queries here:
min-width 760px — show full
min-width 500px — show shortened if hex length > 60 (for txHashes)
< 500px — show shortened

Reply:
Nice idea